<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WLAN Hosted Network Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hostednetwork-gui.qrc">
    <normaloff>:/images/wifi.png</normaloff>:/images/wifi.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="ssidLabel">
      <property name="text">
       <string>SSID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="ssidLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="passphraseLabel">
      <property name="text">
       <string>Passphrase:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passphraseLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="showPassphraseCheckBox">
      <property name="text">
       <string>Show Passphrase</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="logTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>&amp;Application</string>
    </property>
    <addaction name="actionMinimizeToTray"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>&amp;Network</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionShow"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuNetwork"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionShow"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="hostednetwork-gui.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="hostednetwork-gui.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="hostednetwork-gui.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Sto&amp;p</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="hostednetwork-gui.qrc">
     <normaloff>:/images/show.png</normaloff>:/images/show.png</iconset>
   </property>
   <property name="text">
    <string>Sho&amp;w</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="hostednetwork-gui.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="hostednetwork-gui.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About ...</string>
   </property>
  </action>
  <action name="actionMinimizeToTray">
   <property name="text">
    <string>Minimize to &amp;Tray ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ssidLineEdit</tabstop>
  <tabstop>passphraseLineEdit</tabstop>
  <tabstop>showPassphraseCheckBox</tabstop>
  <tabstop>logTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="hostednetwork-gui.qrc"/>
 </resources>
 <connections/>
</ui>
